lib/serial_terminal.pm

```diff
@@ -339,7 +339,7 @@ sub select_serial_terminal {
         } else {
             $console = $root ? 'root-virtio-terminal' : 'user-virtio-terminal';
         }
-    } elsif (get_var('SUT_IP')) {
+    } elsif (get_var('SUT_IP') || get_var('ZVM_HOST')) {
```

Comment:
This should be changed to

    } elsif (get_var('SUT_IP') || is_backend_s390x) {